apps/functions-subscription/src/domain/activation.ts

```diff
@@ -0,0 +1,59 @@
+import * as TE from 'fp-ts/TaskEither';
+import * as t from 'io-ts';
+import { IsoDateFromString } from '@pagopa/ts-commons/lib/dates';
+import { NonNegativeNumber } from '@pagopa/ts-commons/lib/numbers';
+import { TrialIdCodec, UserIdCodec } from './subscription';
+
+const BaseActivationItemCodec = t.strict({
+  id: t.string,
```

Comment:
**question**: Do you think it might be helpful to have specific IDs, like `UserId` or `TrialId`, for `ActivationJob` and `ActivationRequest` items too?